### PR TITLE
[doc]cluster snapshot

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
add cluster snapshot doc
update TOC
code pr #1199

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# 集群快照
+
+## 创建快照
+
+`CREATE SNAPSHOT` 命令可对整个集群创建当前时间点的快照，快照名称由 meta server 的时间戳组成。在创建过程中可能会创建失败，当前版本不支持对创建失败的快照进行自动垃圾回收，后续将在 meta Server 中开发 cluster checker 功能，通过异步线程检查集群状态，并自动回收创建失败的快照垃圾文件。当前版本如果快照创建失败，必须通过 `DROP SNAPSHOT` 命令清除无效的快照。
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md]: Please ensure meta server in consistency

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+-----------------------------------------------------------
+| SNAPSHOT_2019_12_04_10_54_44 | VALID  | 127.0.0.1:77833 |
+-----------------------------------------------------------
+```
+
+Now the deletes snapshot is not in the show snapshots list.
+
+## Tips
+
+- When the system structure changes, it is better to create a snapshot immediately. For example, when you add host, drop host, create space, drop space or balance the data.
```
Comment (bright-starry-sky) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md]: `the data` need to remove. only `balance` is well .

**Comment by bright-starry-sky** [comment]
I have a small question. Are the functions and purposes of snapshots described in other documents? If not, I suggest giving a description.

**Comment by Amber1990Zhang** [comment]
> I have a small question. Are the functions and purposes of snapshots described in other documents? If not, I suggest giving a description.

Thanks for your kind suggestion, but to me, the manual only cares about the operations. Maybe we can post a blog about how the function is worked in social media later. @whitewum

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# 集群快照
+
+## 创建快照
+
+`CREATE SNAPSHOT` 命令可对整个集群创建当前时间点的快照，快照名称由 meta server 的时间戳组成。在创建过程中可能会创建失败，当前版本不支持对创建失败的快照进行自动垃圾回收，后续将在 meta server 中开发 cluster checker 功能，通过异步线程检查集群状态，并自动回收创建失败的快照垃圾文件。当前版本如果快照创建失败，必须通过 `DROP SNAPSHOT` 命令清除无效的快照。
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md]: “”在创建过程中可能会创建失败，当前版本不支持对创建失败的快照进行自动垃圾回收，后续将在 meta server 中开发 cluster checker 功能，通过异步线程检查集群状态，并自动回收创建失败的快照垃圾文件。当前版本如果快照创建失败，必须通过 DROP SNAPSHOT 命令清除无效的快照。

当前版本不支持对指定的图空间创建快照，“  

这段话太啰嗦了，精简一下。有啥注意事项放到末尾去。

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+-----------------------------------------------------------
+| SNAPSHOT_2019_12_04_10_54_42 | VALID  | 127.0.0.1:77833 |
+-----------------------------------------------------------
+| SNAPSHOT_2019_12_04_10_54_44 | VALID  | 127.0.0.1:77833 |
+-----------------------------------------------------------
+```
+
+## 删除快照
+
+`DROP SNAPSHOT` 命令可删除指定名称的快照，语法为 `DROP SNAPSHOT $snapshot-name`，可以通过 `SHOW SNAPSHOTS` 命令获取快照名称，`DROP SNAPSHOT` 既可以删除有效的快照，也可以删除创建失败的快照。
```
Comment (whitewum) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/cluster-snapshot.md]: 语法一般单独一行

**Pull request merged**